nym-vpn-desktop/scripts/appimage.sh

```diff
@@ -71,23 +71,59 @@ tilded() {
 need_cmd mktemp
 temp_dir=$(mktemp -d)
 data_home=${XDG_DATA_HOME:-$HOME/.local/share}
+state_home=${XDG_STATE_HOME:-$HOME/.local/state}
 install_dir="$HOME/.local/bin"
 icons_dir="$data_home/icons"
 appimage="nym-vpn_${version}.AppImage"
 target_appimage="nym-vpn.appimage"
 desktop_dir="$data_home/applications"
+wrapper="nym-vpn-wrapper.sh"
+policy="net.nymtech.nymvpn.policy"
 
+### desktop entry ###
 desktop_entry="[Desktop Entry]
 Name=NymVPN
 Type=Application
 Version=1.0
 Comment=Decentralized, mixnet, and zero-knowledge VPN
-Exec=$install_dir/$target_appimage
+Exec=$install_dir/$wrapper %U
 Icon=$icons_dir/nym-vpn.svg
 Terminal=false
 Categories=Network;"
+###
 
+### app icon ###
 icon='<svg width="32" height="32" viewBox="0 0 32 32" fill="#FB6E4E" xmlns="http://www.w3.org/2000/svg"><path d="M3.7229 29.9997C-0.460546 26.7617 -1.23766 20.7391 2.0003 16.5557C5.23826 12.3722 11.2609 11.5951 15.4443 14.8331C19.6278 18.0711 20.4049 24.0937 17.1669 28.2771C13.9289 32.4605 7.90634 33.2377 3.7229 29.9997ZM28.0076 23.2647C33.3308 17.9415 33.3308 9.31561 28.0076 3.9924C22.6844 -1.3308 14.0455 -1.3308 8.73526 3.9924C3.42501 9.31561 3.41205 17.9415 8.73526 23.2647C14.0585 28.5879 22.6844 28.5879 28.0076 23.2647Z" fill="#FB6E4E"/></svg>'
+###
+
+### wrapper script ###
+wrapper_script="#!/bin/bash
+
+pkexec $install_dir/$target_appimage >$state_home/nym-vpn/vpn.log 2>&1"
+###
+
+### polkit action ###
+polkit_action="<?xml version=\"1.0\" encoding=\"UTF-8\"?>
+<!DOCTYPE policyconfig PUBLIC
+ \"-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN\"
+ \"http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd\">
+<policyconfig>
+ <vendor>Nym</vendor>
+ <vendor_url>https://nymvpn.com</vendor_url>
+ <icon_name>nym-vpn</icon_name>
+ <action id=\"net.nymtech.nymvpn\">
+ <description>NymVPN</description>
+ <message>NymVPN requires root privileges for tunnel purposes</message>
```

Comment:
"NymVPN requires root privileges to create a local virtual network device and set system routing rules"